sw/airborne/modules/ctrl/eff_scheduling_rot_wing.c

```diff
@@ -411,7 +428,7 @@ void stabilization_indi_set_wls_settings(void)
       u_pref_stab_indi[i] = act_pref[i];
       if (i == 5) {
         u_pref_stab_indi[i] = actuator_state_filt_vect[i]; // Set change in prefered state to 0 for elevator
-        u_min_stab_indi[i] = 0; // cmd 0 is lowest position for elevator
+        u_min_stab_indi[i] = 800; // cmd 0 is lowest position for elevator
```

Comment:
Is this for all RotWings? Can I make a parameter out of it?

Reply:
This was changed to get the elevator to a flatter orientation quicker, by limiting the angle it can make to begin with. However, given the better results we get with the changed gains I think we can set this back to zero to confirm we still get good results in the transition.